pkg/resources/management.cattle.io/v3/projectroletemplatebinding/ProjectRoleTemplateBinding.md

```diff
@@ -13,6 +13,7 @@ Users cannot create ProjectRoleTemplateBindings that violate the following const
     - Provided as a non-empty value
     - Specified using the format of `cluster-id:project-id`
     - `project-id`, in particular, must match the namespace of the ProjectRoleTemplateBinding
```

Comment:
nit: 
We don't have to now, but we should change this in the future from cluster-id to cluster.metadata.name as well

Reply:
I've made a change here, though I changed up the wording a bit. If the new wording doesn't work for you let me know.